### PR TITLE
[3.11] gh-103606: Improve error message from logging.config.FileConfig (GH-103628)

### PR DESCRIPTION
**Pull request by Agent-Hellboy**
(cherry picked from commit 152227b569c3a9b87fe0483706f704762ced6d75)

* Issue: gh-103606

----
:books: Documentation preview :books:: https://cpython-previews--104687.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/logging.config.rst [hunk 1]
```diff
@@ -126,6 +130,10 @@ in :mod:`logging` itself) and defining handlers which are declared either in
     .. versionadded:: 3.10
        The *encoding* parameter is added.
 
+    .. versionadded:: 3.12
```
Comment (Agent-Hellboy) [inline Doc/library/logging.config.rst]: I guess we need to change 3.12 to 3.11

Comment (gpshead) [reply Doc/library/logging.config.rst]: I used versionchanged:: 3.11.4

**Review by gpshead — CHANGES_REQUESTED** [review]
I wouldn't merge this in 3.11 yet, there are some things to cleanup in main from the original PR first.

**Comment by Agent-Hellboy** [comment]
>I wouldn't merge this in 3.11 yet, there are some things to cleanup in main from the original PR first.

what are those things?

**Comment by Agent-Hellboy** [comment]
I have added a PR, #104701

**Comment by gpshead** [comment]
> Hi @gpshead I have added a PR, https://github.com/python/cpython/pull/104701

merged.  feel free to just apply those edits to this PR as the backport.

**Comment by Agent-Hellboy** [comment]
I have made the requested changes; please review again

**Review by gpshead — APPROVED** [review]

**Pull request merged**